src/lib/parts.ts

```diff
@@ -69,25 +70,32 @@ export class AttributeCommitter {
   protected _getValue(): unknown {
     const strings = this.strings;
     const l = strings.length - 1;
-    let text = '';
-
-    for (let i = 0; i < l; i++) {
-      text += strings[i];
-      const part = this.parts[i];
-      if (part !== undefined) {
-        const v = part.value;
-        if (isPrimitive(v) || !isIterable(v)) {
-          text += typeof v === 'string' ? v : String(v);
-        } else {
-          for (const t of v) {
-            text += typeof t === 'string' ? t : String(t);
+
+    if (l === 1 && strings[0] === '' && strings[1] === '' &&
```

Comment:
This is pretty similar to https://github.com/Polymer/lit-html/pull/750/files#diff-63a9df26ecde05415a902b5b5dc5e266R136

I think this part won't be necessary once that lands.